extension.js

```diff
@@ -168,6 +179,41 @@ const ExtensionLayout = new Lang.Class({
     GLib.spawn_command_line_async(cmd);
   },
 
+  _findNewStreamerEntries:function(lastList, currentList, detectGameChange) {
+    detectGameChange = detectGameChange || false;
+    if ( lastList.length == 0 )
+      return currentList;
+
+    let streamers = new Map();
```

Comment:
Yeah, a Map. That's what I meant. I think that makes to code cleaner and faster (not that the latter is a problem).